guevaraTorres/assignments/slidingwindow/lc53/lc53.js

```diff
@@ -1,7 +1,53 @@
+/*
+
+each window is size K
+conditon to determine if there's enough elements
+remove the first element in the subarray
+add the new included element in the sliding window
```

Comment:
For psuedo-coding your high-level approach, it would be better to have complete sentences. This way if I am new and reading your code, I can understand what you are attempting to do.

Reply:
Thank you for the feedback, I will keep that in mind.